turbinia/turbiniactl.py

```diff
@@ -45,6 +45,7 @@ def main():
   # TODO(aarontp): Allow for single run mode when
   # by specifying evidence which will also terminate the task manager after
   # evidence has been processed.
+  print("Turbinia version: ", __version__)
```

Comment:
- Could we move this down to after the parsing code (around like 226)?
- Please use single quotes to match the quote style in the rest of the file
- Can we use `log.info()` instead of print?  That way this will also go into the log files.
- Also to match style, can we use the string format method here?  e.g. `'version: {0:s}'.format(__version__)`

Reply:
Sure.